customer_gateway.go

```diff
@@ -10,6 +10,7 @@ type CustomerGateway struct {
 }
 
 // Create creates a new customer from the passed in customer object.
+// You can create a customer by itself, with a payment method, or with a credit card with a billing address.
```

Comment:
I think we should leave this line out as it describes details covered by the Braintree official documentation.